.github/workflows/release.yml

```diff
@@ -31,26 +31,30 @@ jobs:
     needs:
       - create-release
     strategy:
+      fail-fast: false
       matrix:
         include:
           - target: x86_64-unknown-linux-gnu
             os: ubuntu-latest
+          - target: x86_64-unknown-linux-musl
+            os: ubuntu-latest
           - target: x86_64-apple-darwin
```

Comment:
Perhaps it's a good idea to support universal-apple-darwin (I know there's no official name for this, but binstall does use that name) by using lipo

Reply:
Did you mean to just add universal-apple-darwin to the matrix? Or did you mean to replace aarch64-apple-darwin and x86_64-apple-darwin with universal-apple-darwin?

I added universal binary support to upload-rust-binary-action, so either way is ok.

Reply:
> I added universal binary support to upload-rust-binary-action, so either way is ok.

Yes, this was what I am thinking ahout.

Reply:
@NobodyXu 
> > I added universal binary support to upload-rust-binary-action, so either way is ok.
> 
> Yes, this was what I am thinking ahout.

Um, I guess you quoted the wrong sentence?

Reply:
I would guess @NobodyXu meant universal-apple-darwin just from context :thinking:

Reply:
Oh yes, I didn't receive nofitication for @taiki-e 's reply.

Reply:
When I saw NobodyXu's review, I wasn't sure if NobodyXu meant "distributing aarch64 and x86_64 while also distributing universal" or "distributing universal instead of aarch64 and x86_64", so my question was to know which NobodyXu meant.

Reply:
I meant distributing aarch64 and x86_64 while also distributing universal binaries.

Having all available is useful since aarch64 and x86_64 are smaller than universal binary, but users might want universal binary so that it works on any mac target.

Reply:
Thanks for the clarification. I added universal-apple-darwin to the matrix.